lib/controller/page/type/page.summary/page.summary.controller.js

```diff
@@ -139,6 +141,19 @@ SummaryController.attachEmailSubmissions = async (submissionId, userData, submis
   }
 }
 
+SummaryController.attachPdfSubmission = async (submissions, userData, submissionId) => {
+  const title = submissionId
+  const heading = getInstanceProperty('service', 'pdfHeading')
+  const subHeading = getInstanceProperty('service', 'pdfSubHeading')
+  const destination = 'team'
```

Comment:
'team' is hardcoded for now.  Picking this up in a following PR.
Probably need to loop over 'team' and 'user' and add 2 PDF payloads on each submit.